python/spear/unreal_service.py

```diff
@@ -472,8 +472,8 @@ def new_object(self, class_name, outer=0, name="", object_flags=["RF_NoFlags"],
     def load_object(self, class_name, outer, name="", filename="", load_flags=["LOAD_None"], sandbox=0, instancing_context=0):
         return self._entry_point_caller.call("unreal_service.load_object", class_name, outer, name, filename, load_flags, sandbox, instancing_context)
 
-    def load_class(self, class_name, outer, name="", filename="", load_flags=["LOAD_None"], sandbox=0):
-        return self._entry_point_caller.call("unreal_service.load_object", class_name, outer, name, filename, load_flags, sandbox)
+    def load_class(self, class_name, outer, name="", filename="", load_flags=["LOAD_None"], sandbox=0,instancing_context=0):
```

Comment:
As far as I can tell, the `LoadClass<...>(...)` function in Unreal does not accept an instancing context pointer like `LoadObject<...>(...)` does.